### PR TITLE
implement add and remove functionality

### PR DESCRIPTION
**Pull request by ogiogiovictor**
# Changes
1. Implement Redux Add and Remove Functionality

### CODE REVIEW

File: src/components/container/main-content.js [hunk 1]
```diff
@@ -1,44 +1,26 @@
...
 
-  const addBook = (book) => {
-    setBook((previousBook) => [book, ...previousBook]);
-  };
+  // Using Use Selector to grab the state
+  const allbooks = useSelector((state) => state.books);
+
+  // const addBook = (book) => {
+  //   setBook((previousBook) => [book, ...previousBook]);
+  // };
```
Comment (Davidosky007) [inline src/components/container/main-content.js]: * KIndly remove all commented codes from your codebase as it makes your code look untidy and unprofessional 🙏

File: src/components/newbook/addbook.js [hunk 2]
```diff
@@ -1,26 +1,85 @@
...
+
+  const authorChangeHandler = (e) => {
+    setEnteredAuthor(e.target.value);
+  };
+
+  const addNewBook = (e) => {
+    e.preventDefault();
+    // const { target } = e;
+    // const title = target.title.value;
+    // const author = target.author.value;
```
Comment (Davidosky007) [inline src/components/newbook/addbook.js]: * Kindly remove all commented codes as mentoned above 🙏

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @ogiogiovictor 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Well written readme file 👍 
* Proper use of Gitflow 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zhadier — APPROVED** [review]
## Hi @ogiogiovictor  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



# Congratulations! 🎉 

### Keep Rocking!👏👏👏
## Optional
<hr/>

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**